### ISSUE TITLE
Add module name/description tags to pom files

### ISSUE THREAD

**Issue by mandy-chessell** [odpi/egeria#1598]
The following POM files are missing their name and description tags:

- data-platform-services
- data-platform-services-api
- data-platform-services-server
- ranger-connector
- security-officer-connectors
- security-officer-services
- security-officer-services-api
- security-officer-tag-connector
- virtualization-services-api

The snippet below is an example of these tags

```
    <name>OMRS REST API Connector</name>
    <description>
        Uses the OMRS REST API to call an open metadata compliant repository.
    </description>
```

**Comment by planetf1** [comment]
i think this can be closed?

**Comment by innerTide** [comment]
For virtualization-services-api, yes. :)

**Comment by planetf1** [comment]
My error - I misread - whilst some changes are done, there are more to do.

**Comment by cong78** [comment]
Hi @danielaotelea could you please have a quick fix on the pom so this issue can be closed before 1.2 release?

**Comment by cong78** [comment]
All of them have been fixed by adding name and description. So I will close this issue.
